mobile/src/calendar/calendar-actions-button-group.tsx

```diff
@@ -74,7 +73,10 @@ const mapStateToProps = (state: AppState) => ({
 });
 
 const mapDispatchToProps = (dispatch: Dispatch<CalendarActions>) => ({
-    editSickLeave: () => { dispatch(editSickLeave()); }
+    sickLeaveActions: {
+        claim: () => { dispatch(claimSickLeave()); },
```

Comment:
are {} really needed?

Reply:
I think it looks convenient with spread operator